### DISCUSSION TITLE
Map Layer

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#6221] [General]
Can we change the map layer to satellite ? 

I mean from this view 
![Capture](https://user-images.githubusercontent.com/106697208/172394285-bb5471a6-18ba-4e73-a3d7-9f5dbcf0566e.PNG)

to this view 
![Capture1](https://user-images.githubusercontent.com/106697208/172394337-9656db4b-9b67-463d-b9e3-f3b84759b168.PNG)

**Comment by melohagan** [comment]
Hi @rafikApp 

You can change the map tileset URL: https://docs.budibase.com/docs/map#changing-the-map-tileset

This StackOverflow post may be useful for using Google Maps: https://stackoverflow.com/questions/23017766/google-maps-tile-url-for-hybrid-maptype-tiles

**Reply by rafikApp** [reply]
Thank you it works.
I have another problem.
i still want to increase the zoom, but i can't 
![Capture](https://user-images.githubusercontent.com/106697208/172407002-ea3c6133-b5c6-49a6-abfb-6c3efdf8a53e.PNG)

Any solution ?

**Reply by melohagan** [reply]
Have you played around with any of these settings?

![Screenshot 2022-06-07 at 15 39 16](https://user-images.githubusercontent.com/101575380/172408464-6725062c-c1d7-4e1d-b8eb-7eeae07dd6d2.png)

**Reply by rafikApp** [reply]
Yes 
![Capture2](https://user-images.githubusercontent.com/106697208/172409185-e03668dd-3cf2-4a04-9ae2-97a0c542f013.PNG)

**Reply by melohagan** [reply]
This appears to be a limitation with the current zoom values for different tile sets.
I've raised a feature request here: https://github.com/Budibase/budibase/issues/6223

**Reply by rafikApp** [reply]
Ok Thank You
